src/components/ebay-menu-button/template.marko

```diff
@@ -15,35 +15,54 @@
     w-on-expander-expand="handleExpand"
     w-on-expander-collapse="handleCollapse"
     ${processHtmlAttributes(data)}>
-    <ebay-button
-        w-id="button"
-        class=[
-            "${baseClass}__button",
-            (data.borderless && 'expand-btn--borderless')
-        ]
-        variant="expand"
-        size=data.size
-        priority=data.priority
-        no-text=(!data.text && !data.icon)
-        aria-expanded="false"
-        aria-haspopup="true"
-        aria-label=data.a11yText
-        disabled=data.disabled
-        w-on-button-escape="handleButtonEscape">
-        <span class=['expand-btn__cell', (data.label && 'menu-button__control--custom-label')]>
-            <span if(data.label) w-preserve-body>
-                <span w-body=data.label.renderBody body-only-if(true)/>
+    <if(data.variant === 'overflow')>
+        <ebay-button
+            w-id="button"
+            class=[
+                "${baseClass}__button",
+                "expand-btn"
+            ]
+            style="border:0;"
```

Comment:
I was initially concerned about this, but I see why you did this.

I guess the alternative was to add a special class `expand-btn--overflow` and create a new `style.less` file with that class and the `border: 0;` rule.

I'm not opposed to this given the desire to re-use as much as possible. This uses a lot less code and limits payload (albeit, it wouldn't have been a huge amount anyway).